src/components/ButtonPanel.js

```diff
@@ -1,40 +1,73 @@
+import PropTypes from 'prop-types';
 import React from 'react';
 import Button from './Button';
 
-const ButtonPanel = () => {
-  return (
-    <div>
-      <div className="container">
-        <Button btnLabel="AC" />
-        <Button btnLabel="+/-" />
-        <Button btnLabel="%" />
-        <Button btnLabel="/" color="orange" />
-      </div>
-      <div className="container">
-        <Button btnLabel="7" />
-        <Button btnLabel="8" />
-        <Button btnLabel="9" />
-        <Button btnLabel="X" color="orange" />
-      </div>
-      <div className="container">
-        <Button btnLabel="4" />
-        <Button btnLabel="5" />
-        <Button btnLabel="6" />
-        <Button btnLabel="-" color="orange" />
-      </div>
-      <div className="container">
-        <Button btnLabel="1" />
-        <Button btnLabel="2" />
-        <Button btnLabel="3" />
-        <Button btnLabel="+" color="orange" />
-      </div>
-      <div className="container">
-        <Button btnLabel="0" wide />
-        <Button btnLabel="." />
-        <Button btnLabel="=" color="orange" />
+class ButtonPanel extends React.Component {
+  handleClick(buttonName) {
+    // eslint-disable-next-line react/destructuring-assignment
+    this.props.onClick(buttonName);
+  }
+
+  // eslint-disable-next-line max-lines-per-function
+  render() {
+    return (
+      <div>
+        <div className="container">
+          <Button btnLabel="AC" onClick={() => this.handleClick('AC')} />
+          <Button btnLabel="+/-" onClick={() => this.handleClick('+/-')} />
+          <Button btnLabel="%" onClick={() => this.handleClick('%')} />
+          <Button
+            btnLabel="/"
+            color="orange"
+            onClick={() => this.handleClick('/')}
+          />
+        </div>
+        <div className="container">
+          <Button btnLabel="7" onClick={() => this.handleClick('7')} />
+          <Button btnLabel="8" onClick={() => this.handleClick('8')} />
```

Comment:
We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.

This reduces the amount of repetitive code and solves the linter error.